src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/TseClaimantRepReplyService.java

```diff
@@ -0,0 +1,366 @@
+
+        TseRespondType response = TseRespondType.builder()
+                .response(caseData.getTseResponseText())
+                .supportingMaterial(caseData.getTseResponseSupportingMaterial())
+                .hasSupportingMaterial(caseData.getTseResponseHasSupportingMaterial())
+                .from(RESPONDENT_TITLE)
+                .date(UtilHelper.formatCurrentDate(LocalDate.now()))
+                .copyToOtherParty(caseData.getTseResponseCopyToOtherParty())
+                .copyNoGiveDetails(caseData.getTseResponseCopyNoGiveDetails())
+                .build();
+
+        respondCollection.add(TseRespondTypeItem.builder().id(UUID.randomUUID().toString()).value(response).build());
+
+        if (featureToggleService.isWorkAllocationEnabled()) {
+            response.setDateTime(Helper.getCurrentDateTime()); // for Work Allocation DMNs
+            response.setApplicationType(genericTseApplicationType.getType()); // for Work Allocation DMNs
+        }
+
+        if (isRespondingToTribunal) {
+            genericTseApplicationType.setRespondentResponseRequired(NO);
+        }
+
+        genericTseApplicationType.setResponsesCount(String.valueOf(respondCollection.size()));
+    }
+
+    /**
+     * Send emails when LR submits response to Tribunal request/order.
+     */
+    public void sendRespondingToTribunalEmails(CaseDetails caseDetails, String userToken) {
+        sendEmailToTribunal(caseDetails);
+        sendEmailToRespondentForRespondingToTrib(caseDetails);
+        sendAcknowledgementEmailToLR(caseDetails, userToken, true);
+    }
+
+    private void sendEmailToTribunal(CaseDetails caseDetails) {
+        CaseData caseData = caseDetails.getCaseData();
+        String email = claimantTseService.getTribunalEmail(caseData);
+
+        if (isNullOrEmpty(email)) {
+            return;
+        }
+
+        GenericTseApplicationType selectedApplication = getClaimantRepSelectedApplicationType(caseData);
+        Map<String, String> personalisation = Map.of(
+                CASE_NUMBER, caseData.getEthosCaseReference(),
+                APPLICATION_TYPE, selectedApplication.getType(),
+                LINK_TO_EXUI, emailService.getExuiCaseLink(caseDetails.getCaseId()));
+        emailService.sendEmail(replyToTribunalEmailToTribunalTemplateId, email, personalisation);
+    }
+
+    private void sendEmailToRespondentForRespondingToTrib(CaseDetails caseDetails) {
+        CaseData caseData = caseDetails.getCaseData();
+
+        if (!YES.equals(caseData.getTseResponseCopyToOtherParty())) {
+            return;
+        }
+
+        List<String> respondentEmailAddressList = getRespondentEmailAddressList(caseData);
+        if (respondentEmailAddressList.isEmpty()) {
+            return;
+        }
+
+        Map<String, String> personalisation = Map.of(
+                CASE_NUMBER, caseData.getEthosCaseReference(),
+                LINK_TO_CITIZEN_HUB, emailService.getCitizenCaseLink(caseDetails.getCaseId()));
```

Comment:
Respodent reps should be getting a link to ExUI and not citizen hub

Reply:
addressed in [088840c](https://github.com/hmcts/et-ccd-callbacks/pull/2259/commits/088840cf6cb992ae2b002e69019b5941c899201b)